services-manager/src/main/resources/application-example.yaml

```diff
@@ -0,0 +1,83 @@
+# Copyright 2020 Crown Copyright
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+app:
+  version: '@project.version@'
+# While the library happens to be the same version as services, it should be declared here as a static name
+library:
+  version: ${app.version}
+
+manager:
```

Comment:
See comment as to `application.yaml`
But also, why reiterate the jar path when it's already in the default yaml?

Reply:
Happy to have example profile, but cot down on duplication - we only really need the loader path stuff